From c2f74c5810aa5f88cd44c29449a53b68850b8472 Mon Sep 17 00:00:00 2001
From: LuK1337 <priv.luk@gmail.com>
Date: Thu, 30 May 2024 10:52:43 +0200
Subject: [PATCH 2/2] Disable publication signing

---
 wrappers/android/zxingcpp/build.gradle.kts | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/wrappers/android/zxingcpp/build.gradle.kts b/wrappers/android/zxingcpp/build.gradle.kts
index 1f52ebb7..26e82cb8 100644
--- a/wrappers/android/zxingcpp/build.gradle.kts
+++ b/wrappers/android/zxingcpp/build.gradle.kts
@@ -114,10 +114,3 @@ publishing {
         }
     }
 }
-
-signing {
-    val signingKey: String? by project
-    val signingPassword: String? by project
-    useInMemoryPgpKeys(signingKey, signingPassword)
-    sign(publishing.publications)
-}
-- 
2.34.1

